tests/unit/bokeh/test_model.py

```diff
@@ -141,6 +141,20 @@ def test_attr_selector_creates_customjs_int(self) -> None:
         assert cb.args == dict(other=m2)
         assert cb.code == "other.b = this.a[1]"
 
+    def test_attr_selector_creates_customjs_with_zero(self) -> None:
```

Comment:
Fine to merge this as-is later today but just for your FYI situations like this with a lot of repeated boilerplate test code are a good use case for `@pytest.mark.parametrize`

Reply:
👍 I'll have to familiarize myself a lot more with pytest